sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkBatchClient.cs

```diff
@@ -28,5 +31,145 @@ public SparkBatchClient(Uri endpoint, string sparkPoolName, TokenCredential cred
+        {
+            using var scope = _clientDiagnostics.CreateScope("SparkBatchClient.CreateSparkBatchJob");
+            scope.Start();
+            try
+            {
+                return await RestClient.CreateSparkBatchJobAsync(sparkBatchJobOptions, detailed, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Create new spark batch job. </summary>
+        /// <param name="sparkBatchJobOptions"> Livy compatible batch job request payload. </param>
+        /// <param name="detailed"> Optional query param specifying whether detailed response is returned beyond plain livy. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        internal virtual Response<SparkBatchJob> CreateSparkBatchJob(SparkBatchJobOptions sparkBatchJobOptions, bool? detailed = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("SparkBatchClient.CreateSparkBatchJob");
+            scope.Start();
+            try
+            {
+                return RestClient.CreateSparkBatchJob(sparkBatchJobOptions, detailed, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        public virtual async Task<SparkBatchOperation> StartCreateSparkBatchJobAsync(SparkBatchJobOptions sparkBatchJobOptions, bool? detailed = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("SparkBatchClient.StartCreateSparkBatchJobAsync");
+            scope.Start();
+            try
+            {
+                Response<SparkBatchJob> batchSession = await RestClient.CreateSparkBatchJobAsync(sparkBatchJobOptions, detailed, cancellationToken).ConfigureAwait(false);
+                return new SparkBatchOperation(this, this._clientDiagnostics, batchSession);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        public virtual SparkBatchOperation StartCreateSparkBatchJob(SparkBatchJobOptions sparkBatchJobOptions, bool? detailed = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("SparkBatchClient.StartCreateSparkBatchJob");
+            scope.Start();
+            try
+            {
+                Response<SparkBatchJob> batchSession = RestClient.CreateSparkBatchJob(sparkBatchJobOptions, detailed, cancellationToken);
+                return new SparkBatchOperation(this, this._clientDiagnostics, batchSession);
```

Comment:
nit: I thought we had a stylecop rule for this (maybe something changed) but we generally don't include `this` when referencing instance members.